RUST_CODING_STYLE.md

```diff
@@ -0,0 +1,364 @@
+# Coding Guidelines for Aptos Core
+
+This document describes the coding guidelines for the [Aptos Core](https://github.com/aptos-labs/aptos-core) Rust codebase. For the Move language, see the [Move Coding Conventions](https://aptos.dev/move/book/SUMMARY).
+Secure coding guidance is provided in the [Aptos Rust Secure Coding Guidelines](./RUST_SECURE_CODING.md).
+
+## Code formatting
+
+All code formatting is enforced with [rustfmt](https://github.com/rust-lang/rustfmt) with a project-specific configuration. Below is an example command to adhere to the Aptos Core project conventions.
+
+```
+cargo fmt
+```
+
+## Code analysis
+
+[Clippy](https://github.com/rust-lang/rust-clippy) is used to catch common mistakes and is run as a part of continuous integration. Before submitting your code for review, you can run clippy with our configuration:
+
```

Comment:
Note that there is a script to handle all of these, so are these rust guidelines for this repository or for anything in Rust?

Reply:
I copied this from the actual DOC, probably it wasn't updated. Let me fix this :)